### PR TITLE
Mosviz doc updates

### PR DESCRIPTION
**Pull request by rosteen**
Fleshing out some more Mosviz docs, creating as a draft for now since I'm still trying to figure out why one of my links isn't working.

### CODE REVIEW

**Comment by rosteen** [comment]
Turned out that I just needed to rebuild my docs after a `make clean`, the link was working.

File: docs/mosviz/import_data.rst [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
 Import Data
 ***********
 
-Words go here...
+Currently, data must be imported into Mosviz using the API in the Jupyter notebook. 
+After initializing the app as explained in :ref:`mosviz-notebook`, 
+you must generate three lists containing the filenames for the 1D spectra, 
+2D spectra, and images in your dataset. These three lists are taken as arguments 
+to the Mosviz `load_data` method. An example is given below, where `file_dir` is a 
+directory that contains all the files for the dataset to be loaded.
```
Comment (javerbukh) [inline docs/mosviz/import_data.rst:12]: This line needs to end with a colon in order for the following lines to count as code.

Comment (rosteen) [reply docs/mosviz/import_data.rst]: Just rebased and pushed an update with this change.

**Comment by rosteen** [comment]
@javerbukh It seems like the automated tests fail if you call e.g. `mosviz.app` in a code section. Would going back to the previous version (period instead of colon preceding the code block) lead to the code block not being tested? I'm unsure what the best course of action here is.

**Review by PatrickOgle — APPROVED** [review]
Looks good.   Consider removing the code that is crashing the CI and instead displaying it as non-executable text.

**Comment by javerbukh** [comment]
@rosteen I think what you actually want to do there is `::` instead of `:`. If that does not work, then Patrick's suggestion sounds good to me.

**Comment by rosteen** [comment]
Well, apparently the `>>> mosviz.app` line causes the doctest to fail regardless of what the code block is proceeded by. I think I remember a way to mark a line or block to be ignored by the doctests, I'll look into that shortly.

**Review by javerbukh — APPROVED** [review]
Checked on the RTD page and it looks great!

**Comment by javerbukh** [comment]
Merging.

**Pull request merged**